discover/commerce/articles/01-catalog-management/01-product-categories.markdown

```diff
@@ -0,0 +1,112 @@
+also makes it difficult for administrators to see the entire vocabulary. To
+view all categories at once, go to *Control Panel* &rarr; *Configuration*
+&rarr; *System Settings* &rarr; *Assets* and select *Flattened Tree* from the
+drop-down menu in *Asset Categories Web*. Then return to *Site Menu* &rarr;
+*Content* &rarr; *Categories*.
+
+$$$
+
+## Adding and Removing Products from Categories [](id=adding-and-removing-products-from-categories)
+
+To add a product to one or more categories:
+
+1.  Go to the catalog, select a product, and open its *Categorization* tab.
+
+2.  Click on the *Select* button under the label for the appropriate vocabulary,
+    and in the pop-up that appears the check the box next to each category that
+    applies.
+
+3.  Click Save.
+
+A product can be removed from a category the same way by unchecking the boxes in
+step 2. Alternatively, several products can be removed at once using the category
+application:
+
+1.  Find the category you want at *Site Menu* &rarr; *Content* &rarr;
+    *Categories*. Then click ![Options](../../images/icon-options.png)
+    &rarr; *Edit*, and select the *Products* tab.
+
+2.  Check the box next to each product you want to remove from the category.
+
+3.  Click *Remove*.
+
+## Other Category Associations [](id=other-category-associations)
+
+You may want to associate more than just products with your categories. Creating
+relationships with pages and images can make it easier to display and maintain
+category content.
+
+### Category Display Page [](id=category-display-page)
+
+The category display page is a feature that creates a relationship between
+a category, a specified page, and a URL for that page. This relationship makes
+it easy to display and organize content or applications related to to that
+category, as changes made to the category will be automatically reflected on the
+page. A typical use case might involve using the page to display category
+content for customers.
+
+To create a category display page:
+
+1.  Create a page.
+
+2.  Find the category you want to associate with the page in *Site Menu* &rarr;
+    *Content* &rarr; *Categories*. Then click
+    ![Options](../../images/icon-options.png) &rarr; *Edit*, and select the
+    Category Display Page tab. 
+
+3.  Click *Choose* and select the page you just created.
+
+4.  Change the *Friendly URL* if you want it to be something other than the
+    default.
+
+5.  Click *Done* to close the pop-up and then click *Save*.
+
+This process creates a relationship between the category and a blank page. At
+any point, you can populate the page with whatever content or applications suit
+your purposes.
+
+### Images [](id=images)
+
+You may want to upload images and associate them with product categories. This
+allows you to designate a generic category page that will automatically display
+images and products for whatever category a buyer is searching for. These images
```

Comment:
Not a stickler on not ending a sentence with a preposition, but there's usually a better way to word it. Maybe "...for whatever category a buyer wants."